rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book

```

Comment:
- [ ] Please set the `person` and `book` as an attribute accessor. This will enable us to read and write easily.